test/e2e/specs/site-editor/dataviews-pagination.spec.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.describe( 'DataViews Pagination', () => {
+	test.beforeAll( async ( { requestUtils } ) => {
+		await requestUtils.activateTheme( 'emptytheme' );
+		// Create 11 pages to force pagination when perPage is set to 10.
+		await Promise.all(
+			Array( 11 )
+				.fill()
+				.map( ( _, i ) =>
+					requestUtils.createPage( {
+						title: `Test Page ${ i + 1 }`,
+						status: 'publish',
+					} )
+				)
+		);
```

Comment:
```suggestion
		await requestUtils.batchRest(
			Array( 11 )
				.fill()
				.map( ( _, i ) => ( {
					method: 'POST',
					path: '/wp/v2/pages',
					body: {
						title: `Test Page ${ i + 1 }`,
						status: 'publish',
					},
				} ) )
		);
```

Nit: Using `batchRest` should reduce the number of requests and improve performance.

Reply:
Cool. I didn't know about that util, thanks!